### PR TITLE
Add {{take}} and {{drop}} helpers

### PR DESCRIPTION
**Pull request by martndemus**

### CODE REVIEW

File: tests/integration/helpers/drop-test.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+test('It drops the first N entries of array', function(assert) {
+  this.set('array', emberArray([1, 2, 3, 4, 5]));
+
+  this.render(hbs`
+    {{~#each (drop array 2) as |n|~}}
+      {{n}}
+    {{~/each~}}
+  `);
+
+  assert.equal(this.$().text().trim(), '345');
```
Comment (poteto) [inline tests/integration/helpers/drop-test.js]: Assertion messages please! 😬

File: tests/integration/helpers/take-test.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+test('It takes the first N entries of array', function(assert) {
+  this.set('array', emberArray([1, 2, 3, 4, 5]));
+
+  this.render(hbs`
+    {{~#each (take array 2) as |n|~}}
+      {{n}}
+    {{~/each~}}
+  `);
+
+  assert.equal(this.$().text().trim(), '12');
```
Comment (poteto) [inline tests/integration/helpers/take-test.js]: Assertion messages please! 🍐

File: app/helpers/drop.js [hunk 3]
```diff
@@ -0,0 +1 @@
+export { default, drop } from 'ember-functional-helpers/helpers/drop';
```
Comment (poteto) [inline app/helpers/drop.js]: Can we also export in https://github.com/DockYard/ember-functional-helpers/blob/remove-compose/addon/index.js please

**Comment by martndemus** [comment]
Closing in favor of #11

**Pull request closed without merging**